components/vf-sass-config/variables/vf-global-variables.scss

```diff
@@ -8,7 +8,7 @@ $global-font-family: $vf-font-family--sans-serif !default;
 $vf-text-margin--bottom: 16px !default;
 
 // grid variables
-$global-grid-column-gap: 1em !default;
+$global-column-gap: 1em !default;
```

Comment:
I don't think most use cases would require it but also add 

```
$global-grid-column-gap: 1em !default; // legacy support, will be removed in v1.some release
```

Will give us insurance if an older component needs to be used for some reason

Reply:
Dang, I was trying not to effect the Sass variables at all. I’m not a fan of how the CSSWG has moved towards a single rule that effects both grid and flex box, I would have much preferred grid-gap and flex-gap so we can distinguish them - so my hope was to keep Sass variables explicit in what the gap is for … 

So I’ll change this back to `global-grid-column-gap` again